framework/PostProcessors/CrossValidation.py

```diff
@@ -185,6 +187,7 @@ def inputToInternal(self, currentInp, full = False):
       understandable by this pp.
       @ In, currentInp, list or DataObject, data object or a list of data objects
       @ In, full, bool, optional, True to retrieve the whole input or False to get the last element of the input
+        TODO, full should be removed
```

Comment:
Is this a dataobject rework TODO, or a future TODO for devel (in which case we might need an issue for it)

Reply:
I think this is a data object rework TODO.

Reply:
Okay.  What is the change need before it's acted on?  Is this a "once someone has time and needs speed" changes?

Reply:
When we start to finalize the inputToInternal, and remove the collectOutput. The final clean up I think.